{
    "cveId": "CVE-2025-2999",
    "version": "1.0.0",
    "timestamp": "2025-07-04T16:12:41.853043+00:00",
    "description": "A vulnerability was found in PyTorch 2.6.0. It has been rated as critical. Affected by this issue is the function torch.nn.utils.rnn.unpack_sequence. The manipulation leads to memory corruption. Attacking locally is a requirement. The exploit has been disclosed to the public and may be used.",
    "keyphrases": {
        "rootcause": "",
        "weakness": "memory corruption",
        "impact": "",
        "vector": "",
        "attacker": "",
        "product": "PyTorch",
        "version": "2.6.0",
        "component": "torch.nn.utils.rnn.unpack_sequence function"
    }
}
